index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/105bcdf182.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>Hello World</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a class="logo">Habib Payenda</a>
+            <i id="menu" class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <section class="headline">
+        <div class="text-holder">
+            <h1 class="primary-text">I'm Habib glad to see you!</h1>
+            <p class="details">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis deserunt facilis tempore cupiditate. Reprehenderit, voluptatibus nam corrupti omnis quam vero, id minus ea quae in aperiam eum, veritatis aliquid non.
+            </p>
+            <a class="connect">Let's connect</a>
+        </div>
+        <div class="fa-container">
+            <i class="fa-brands fa-twitter fa"></i>
+            <i class="fa-brands fa-linkedin-in fa"></i>
+            <i class="fa-brands fa-medium fa" ></i>
+            <i class="fa-brands fa-github fa"></i>
+            <i class="fa-solid fa-hand-peace fa"></i>
+        </div>
```

Comment:
Let's wrap all the icons in a `ul` tag and make the individual icons list items.

Reply:
OK for sure!